### PR TITLE
Drop EL7 support

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: hooks/pre/32-install_selinux_packages.rb [hunk 1]
```diff
@@ -2,17 +2,12 @@
...
-    packages << 'rh-redis5-redis' if local_redis?
-  end
-
   packages << 'foreman-selinux' if foreman_server?
   packages << 'katello-selinux' if katello_enabled?
   packages << 'candlepin-selinux' if katello_enabled?
   packages << 'pulpcore-selinux' if pulpcore_enabled?
 
-  if katello_enabled? && el8?
+  if katello_enabled?
```
Comment (ekohl) [inline hooks/pre/32-install_selinux_packages.rb]: I think this entire hack is no longer needed now that we have proper DNF modules. That would allow us to get rid of the `el8?` definition too.

File: hooks/pre/32-install_selinux_packages.rb [hunk 2]
```diff
@@ -2,20 +2,10 @@
...
-    packages << 'rh-postgresql12-postgresql-server' if local_postgresql?
-    packages << 'rh-redis5-redis' if local_redis?
-  end
-
   packages << 'foreman-selinux' if foreman_server?
   packages << 'katello-selinux' if katello_enabled?
   packages << 'candlepin-selinux' if katello_enabled?
   packages << 'pulpcore-selinux' if pulpcore_enabled?
 
-  if katello_enabled? && el8?
```
Comment (ekohl) [inline hooks/pre/32-install_selinux_packages.rb]: Perhaps this could be a separate PR and already merged. No need to wait for it IMHO.

Comment (ekohl) [reply hooks/pre/32-install_selinux_packages.rb]: https://github.com/theforeman/foreman-installer/pull/786

**Comment by ehelms** [comment]
The nightly testing pipelines have dropped EL7 testing paths, therefore this should be mergable.

**Review by ekohl — APPROVED** [review]
I think we should also drop the EL7 migrations https://github.com/theforeman/foreman-installer/blob/6be1e967f934aee9d8a9350ab52246c93da60858/config/foreman.migrations/20200113133837_reset_bind-host-foreman-proxy.rb and https://github.com/theforeman/foreman-installer/blob/develop/config/katello.migrations/200113133837-reset-bind-host-foreman-proxy.rb.

**Comment by ehelms** [comment]
> I think we should also drop the EL7 migrations https://github.com/theforeman/foreman-installer/blob/6be1e967f934aee9d8a9350ab52246c93da60858/config/foreman.migrations/20200113133837_reset_bind-host-foreman-proxy.rb and https://github.com/theforeman/foreman-installer/blob/develop/config/katello.migrations/200113133837-reset-bind-host-foreman-proxy.rb.

Dropped like they were hot

**Review by ekohl — APPROVED** [review]
